crates/apollo-mcp-server/src/runtime/telemetry.rs

```diff
@@ -38,67 +39,132 @@ pub struct Exporters {
     tracing: Option<TracingExporters>,
 }
 
+/// Telemetry exporter options
+#[derive(Debug, Deserialize, JsonSchema)]
+#[serde(tag = "protocol", rename_all = "lowercase")]
+pub enum TelemetryExporter {
+    /// GRPC Exporter
+    Grpc {
+        endpoint: String,
+        #[serde(default)]
+        temporality: MetricTemporality,
```

Comment:
Does this field apply only to metrics exporters and not tracing? If that's the case, when the enum is used for tracing, the `temporality` will be ignored silently. 

What do you think about making this field optional and adding runtime validation based on the exporter type? Another option could be to split it into `MetricExporter` and `TracingExporter`.